book/http_cache.rst

```diff
@@ -163,6 +163,23 @@ kernel::
 The caching kernel will immediately act as a reverse proxy - caching responses
 from your application and returning them to the client.
 
+..caution::
+
+    By default, the caching kernel ignores the ``framework.http_method_override``
```

Comment:
I'm not sure about this: `cached kernel` means that the kernel itself is cached. But `caching kernel` means that is the kernel the one who caches things. I think the second definition is more accurate. But I agree that *"caching kernel"* doesn't sound great.